src/render.js

```diff
@@ -1,5 +1,7 @@
 // @flow
 import * as React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { View } from 'react-native';
```

Comment:
I guess it's time to make "react-native" a peer dependency. Making it ">= 0.59" should be fine

Reply:
done